content/en/security/application_security/enabling/nodejs.md

```diff
@@ -151,9 +151,93 @@ DD_APPSEC_ENABLED=true node app.js
 
 {{< img src="/security/application_security/appsec-getstarted-threat-and-vuln.mp4" alt="Video showing Signals explorer and details, and Vulnerabilities explorer and details." video="true" >}}
 
+## Enabling custom code vulnerability detection
+If your service is running a [tracing library version that supports Vulnerability Management for Custom Code Vulnerability Detection][3], enable it by setting the `DD_IAST_ENABLED=true` environment variable and restart your service.
+
+
+To leveraging custom code vulnerability detection capabilities for your service:
+
+1. [Update your Datadog Agent][4] to at least version 7.41.1.
+2. Update your tracing library to at least the minimum version needed to turn on custom code vulnerability detection. For details, see [ASM capabilities support][3].
+3. Add the `DD_IAST_ENABLED=true` environment variable to your application configuration.
+
+If you initialize the APM library on the command line using the `--require` option to Node.js:
+
+ ```shell
```

Comment:
~~~suggestion
   If you initialize the APM library on the command line using the `--require` option to Node.js:

   ```shell
~~~